galleon-pack/galleon-shared/pom.xml

```diff
@@ -365,6 +365,17 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>io.smallrye.reactive</groupId>
+            <artifactId>smallrye-mutiny-vertx-amqp-client</artifactId>
+            <exclusions>
```

Comment:
Future fix:

'exclusions' should not appear here unless there is a good reason they don't appear in the bom.

Reply:
Made a start in https://github.com/wildfly/wildfly/pull/17491